src/extension.ts

```diff
@@ -96,7 +94,16 @@ export async function activate(ctx: vscode.ExtensionContext) {
   ctx.subscriptions.push(watcher);
   await genEnvFile(buildDir);
 
-  let compileCommandsHandler = async () => {
+  // Refresh if the extension configuration is changed. builddir changes won't be reflected, however.
```

Comment:
What would it take to reflect builddir change? That seems to be the most important option that should trigger a refresh.

Reply:
Can be done in separate PR if we have a way of course, not blocking this PR.

Reply:
Well, #103 😀 I am not looking forward to rebasing it...

Maybe for now the simplest option is to reload the whole extension, given how early on `buildDir` is read and cached after activation?

Reply:
> Maybe for now the simplest option is to reload the whole extension, given how early on buildDir is read and cached after activation?

Fair enough, can you open a PR please?